requirements-dev.txt

```diff
@@ -2,7 +2,8 @@ mock==5.1.0
 nose2[coverage_plugin]==0.13.0
 cov-core==1.15.0
 black==22.12.0
-pylint==2.17.6
+pylint==2.17.6; python_version < '3.12'
+pylint==3.0.0a7; python_version >= '3.12'
```

Comment:
FYI pylint 3.0.1 has been merged into master 

- https://github.com/searxng/searxng/pull/2907

Reply:
Looking at the history of pylint, I'm going to way few days for new quick releases.

Reply:
uvloop: new version with Python 3.12 support:
https://twitter.com/1st1/status/1710865834975822112

Reply:
@dalf should we really wait about uvloop? .. I think its time to merge py3.12 support:

@Bnyro wrote: [Important: I did not yet test this because my Linux distro updated to Python 3.12 which we don't yet support, hence I can't build SearXNG on my working machine anymore :/](https://github.com/searxng/searxng/pull/2927)

Reply:
I don't want to confuse speed with haste.
It is on the way: https://github.com/MagicStack/uvloop/pull/570

And uvloop is going to be dropped with the searx.network PR.

If it is urgent, feel free to create another PR or push on top of this one:
* python-version: ["3.8", "3.9", "3.10", "3.11", "3.12.0"] <--- not 3.12.0rc3
* remove pylint change (it is already done)
* keep the uvloop dependency change.

Reply:
@dalf there is an uvloop update --> https://github.com/searxng/searxng/pull/2949